### DISCUSSION TITLE
Comparing ORM energy consumption versus SQL

### DISCUSSION THREAD

**Discussion by pmk7** [mlco2/codecarbon#730] [Q&A]
Hey all,

Has anyone used this tool to measure the energy consumption of an ORM- for example, SQLAlchemy- and compared it to the energy consumption of the same operation in pure SQL? I'm about to start working on a university project and am examining the tools that are out there. Any feedback or insight from someone who has looked into this kind of use case would be really appreciated.

**Comment by benoit-cty** [comment]
Hello,
Nice idea. Let us know what you find.
